src/main/java/baritone/behavior/InventoryBehavior.java

```diff
@@ -167,19 +163,18 @@ public boolean selectThrowawayForLocation(boolean select, int x, int y, int z) {
         if (maybe != null && throwaway(select, stack -> stack.getItem() instanceof ItemBlock && ((ItemBlock) stack.getItem()).getBlock().equals(maybe.getBlock()))) {
             return true;
         }
-        for (Item item : Baritone.settings().acceptableThrowawayItems.value) {
-            if (throwaway(select, stack -> item.equals(stack.getItem()))) {
-                return true;
-            }
-        }
-        return false;
+        return throwaway(select, this::isThrowawayItem);
```

Comment:
This technically changes the behavior of the method by considering the slot before the item, but that's what `firstValidThrowaway` does so /shrug